server/routes/tests/chapterRouter.test.ts

```diff
@@ -0,0 +1,62 @@
+import App from 'server/testUtils/App';
+import chapterRouter from 'server/routes/v1/chapter';
+
+describe('router: sampleRouter', () => {
+  const app = new App();
+
+  beforeAll(async () => {
+    await app.initialize({ withRouter: chapterRouter });
+  });
+
+  describe('/chapter', () => {
+    it('Creates a Chapter', async () => {
+      const location = await app.request.post('/locations').send({
```

Comment:
We need to setup testing DB or maybe mock TypeORM for this. @timmyichen any ideas?

Reply:
We should just have a bunch of helper functions (factories?) to aid with the creation of models we need.

Reply:
Yeah, but there must be an function to remove data after a test

Reply:
tests data should be spun up in a test database anyway (which is nuked/recreated when running tests), is this configurable by typeorm?

Reply:
I was also thinking about (this)[https://gist.github.com/Zeko369/2e81e78d1eca03ca35a5a1fc7d63b222] this is a `rspec` test (for rails controller) where I can
1. easily create entities at the beginning (also there is a factory linked)
2. can check what changed in the db during a (request)[https://gist.github.com/Zeko369/2e81e78d1eca03ca35a5a1fc7d63b222]

I'll look into what can be done with `typeorm` in that regard

Reply:
Resolving this, it'll be done in a different PR